tests/transform/index.js

```diff
@@ -133,10 +80,11 @@ test('emits error for syntax error', function (t) {
   var src = 'var html = require(\'nanohtml\')\n  module.exports = function (data) {\n    var className = (\'test\' + ) // <--- HERE\'S A SYNTAX ERROR\n    return html`<div class="${className}">\n      <h1>${data}</h1>\n    </div>`\n  }' // eslint-disable-line
   fs.writeFileSync(FIXTURE, src)
   var b = browserify(FIXTURE, {
-    browserField: false,
     transform: path.join(__dirname, '../../')
   })
+  // don't b.require createElement, it will hang this test
```

Comment:
oops this is probably this browserify bug: https://github.com/browserify/browserify/issues/1707 looks like that might still be an issue after all